ecl/eclcc/eclcc.cpp

```diff
@@ -1976,7 +1976,12 @@ bool EclCC::parseCommandLineOptions(int argc, const char* argv[])
         else if (strcmp(arg, "-internal")==0)
         {
             outputSizeStmts();
-            testHqlInternals();
+            int error = testHqlInternals() + testReservedWords();  //NOTE: testReservedWords() depends on testHqlInternals() so must be be called after.
+            // report test result
+            if (error)
+                printf("%d error%s found!\n", error, error<=1?"":"s");
+            else
+                printf("No errors\n");
```

Comment:
Still not sure how this works if not failing, given that only a single error value can be returned? Shall I just return there and then and change the use that -internal can be no longer be (transparently (other than screen prints)) used with a normal compilation? @ghalliday any suggestions regarding how you may (if at all) use -internal?